src/taoensso/timbre/appenders/3rd_party/newrelic.clj

```diff
@@ -0,0 +1,21 @@
+(ns taoensso.timbre.appenders.3rd-party.appenders.newrelic
+  "A New Relic appender"
+  {:author "Camilo Polymeris"}
+  (:import (com.newrelic.api.agent NewRelic)
+           (java.util HashMap)))
+
+(defn newrelic-appender
+  "New Relic appender. The Java agent must be installed and configured via newrelic.yml, see
+  https://docs.newrelic.com/docs/agents/java-agent/installation/java-agent-manual-installation#h2-install-agent"
+  [& _]
+  {:enabled?   true
+   :async?     true
+   :min-level  :warn
+   :rate-limit [[100 60000]]                                ;matches New Relic's own cap
+   :output-fn  :inherit
+   :fn         (fn [data]
+                 (let [{:keys [output-fn ?err level]} data
+                       params (HashMap. {"Log level" (name level)})]
+                   (if ?err
+                     (NewRelic/noticeError ?err params)
+                     (NewRelic/noticeError (output-fn data) params))))})
```

Comment:
No problem. It's true that NR is not a logging facility.

That said, for me, it's just practical that at the very least error and fatal should go trough New Relic (and that's what I actually do in my project, not sure why I specified `:warn` as the default). Sure I could call `noticeError` directly, but if I already have the timbre hook, and the cases where I log an error (i.e. when the transaction or job *failed*) are exactly the same I would report to NR... why not use that?